.github/workflows/pylint-test.yml

```diff
@@ -21,7 +21,7 @@ jobs:
     - name: Running unit tests
       run: pytest
     - name: Analysing the code with pylint
-      run: pylint --rcfile=pylint.cfg --recursive=y nmmo tests
+      run: pylint --recursive=y nmmo tests
```

Comment:
was this intentional?

Reply:
Yes, this was intentional.

I renamed pylint.cfg to .pylintrc, so that pylint can find it by default. This allows us to run the same pylint without the --rcfile flag. The `pre-git-check.sh` was also modified, and it works as expected.